tests_external/helpers/index.mjs

```diff
@@ -0,0 +1,26 @@
+import {execSync} from 'child_process';
```

Comment:
It perfectly make sense to move this logic to a common place, but I think it's better to move it to `tools`: https://github.com/GoogleChromeLabs/chromium-bidi/pull/1616